pkg/controller/batchrelease/batchrelease_special_cases_handler.go

```diff
@@ -130,6 +131,18 @@ func (r *Executor) checkHealthBeforeExecution(controller workloads.WorkloadContr
 		reason = "WorkloadNotStable"
 		message = "workload status is not stable, then wait"
 		needStopThisRound = true
+
+	case isWorkloadRollbackInBatch(workloadEvent, r.release, workloadInfo, r.releaseStatus):
```

Comment:
is it possible to merge this case with the one for "isWorkloadChanged" , they both starts with RolloutPhaseProgressing, and had podtemplate changed.